schematic_api/api/openapi/api.yaml

```diff
@@ -101,8 +101,57 @@ paths:
         - Manifest Operations
   /manifest/download:
```

Comment:
If this is in your control, my advise is to redesign the API as such, and to not include `manifestId` in the query parameter.

```
GET /manifest/{manifestId}
```

Reply:
This can be a Jira ticket

Reply:
Yeah thanks Tom. There are a bunch of endpoints that need redesign like this. I open a placeholder ticket: https://sagebionetworks.jira.com/browse/DCA-138